vslm/client.go

```diff
@@ -39,7 +39,8 @@ var (
 )
 
 type Client struct {
-	*soap.Client
+	vim25Client   *vim25.Client
```

Comment:
Looks like this is only referenced by the new test and not used by the vslm package itself, or is there a future use-case you had in mind?

Reply:
Actually I was trying to make vslm client struct look same as cns client. And I forgot to revert my initial changes. It is not needed. Thanks for catching this!